src/api/src/services/BinaryFlashingStrategy/BinaryConfigurator/index.ts

```diff
@@ -0,0 +1,175 @@
+            flags.push(['--domain', 'in_866']);
+            break;
+          case UserDefineKey.REGULATORY_DOMAIN_ISM_2400:
+            break;
+          case UserDefineKey.REGULATORY_DOMAIN_EU_CE_2400:
+            flags.push(['--lbt']);
+            break;
+          case UserDefineKey.AUTO_WIFI_ON_INTERVAL:
+            flags.push(['--auto-wifi', userDefine.value!]);
+            break;
+          case UserDefineKey.HOME_WIFI_SSID:
+            flags.push(['--ssid', userDefine.value!]);
+            break;
+          case UserDefineKey.HOME_WIFI_PASSWORD:
+            flags.push(['--password', userDefine.value!]);
+            break;
+          case UserDefineKey.LOCK_ON_FIRST_CONNECTION:
+            break;
+          case UserDefineKey.JUST_BEEP_ONCE:
+            flags.push(['--buzzer-mode', 'one-beep']);
+            break;
+          case UserDefineKey.DISABLE_ALL_BEEPS:
+            flags.push(['--buzzer-mode', 'quiet']);
+            break;
+          case UserDefineKey.DISABLE_STARTUP_BEEP:
+            flags.push(['--buzzer-mode', 'quiet']);
+            break;
+          case UserDefineKey.MY_STARTUP_MELODY:
+            flags.push(['--buzzer-mode', 'custom']);
+            flags.push(['--buzzer-melody', userDefine.key]);
+            break;
+          case UserDefineKey.RCVR_INVERT_TX:
+            flags.push(['--invert-tx']);
+            break;
+          case UserDefineKey.RCVR_UART_BAUD:
+            flags.push(['--rx-baud', userDefine.value!]);
+            break;
+          case UserDefineKey.TLM_REPORT_INTERVAL_MS:
+            const tlmReportIntervalMs = userDefine.value!.replaceAll('LU', '');
+            flags.push(['--tlm-report', tlmReportIntervalMs]);
+            break;
+          case UserDefineKey.UART_INVERTED:
+            flags.push(['--uart-inverted']);
+            break;
+          case UserDefineKey.UNLOCK_HIGHER_POWER:
+            flags.push(['--unlock-higher-power']);
+            break;
+          case UserDefineKey.USE_R9MM_R9MINI_SBUS:
+            flags.push(['--r9mm-mini-sbus']);
+            break;
+          // not supported
+          case UserDefineKey.DEVICE_NAME:
+            break;
+          default:
+            const exhaustiveCheck: never = userDefine.key;
+            throw new Error(`Unhandled color case: ${exhaustiveCheck}`);
+        }
+      });
+    return flags;
+  }
+
+  async run(
+    firmwareSourcePath: string,
+    hardwareDefinitionsPath: string,
+    firmwareBinaryPath: string,
+    flags: string[][],
+    onUpdate: OnOutputFunc = NoOpFunc
+  ) {
+    this.logger.log('flags', {
+      flags: maskSensitiveFlags(flags),
+      hardwareDefinitionsPath,
+      firmwareBinaryPath,
+    });
+    const binaryConfiguratorArgs = [
+      ...this.stringifyFlags([...flags, ['--dir', hardwareDefinitionsPath]]),
+      firmwareBinaryPath,
+    ];
+    return this.python.runPythonScript(
+      path.join(firmwareSourcePath, 'python', 'binary_configurator.py'),
```

Comment:
This should now be using the flasher.pyz file included in the firmware download zip, all the parameters are the same. flasher.pyz is a packages version of the python folder from the main firmware repo will all it's dependencies included.  Check the PR https://github.com/ExpressLRS/ExpressLRS/pull/2145 (also you should be able to test using the zip from the build for that PR too).